extensions/src/main/scala/pl/touk/nussknacker/extensions/db/pool/DBPoolConfig.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.extensions.db.pool
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+case class DBPoolConfig(driverClassName: String,
```

Comment:
I think better solutions is create companion object with default instead case class with defaults. WDYT?

Reply:
Yes, but DTO - DBPoolConfig shouldn't contain defaults.